### ISSUE TITLE
Create Servo code for Grabber

### ISSUE THREAD

**Issue by WebCoder49** [Brighton-FTC/FTC-2023#31]

**Comment by WebCoder49** [comment]
Opens+Closes

**Comment by sbottingota** [comment]
Sorry for the late notice: I had the code but forgot to notify here.
I think I have done this: https://github.com/Brighton-FTC/FTC-2023/commit/8e45a1b196cdf88feacbdea4b6eb1cd302cebd2f
However, I will try to use one button to toggle open/close instead of having one button for each.

**Comment by sbottingota** [comment]
Changed/fixed a few things:
Made it toggle: https://github.com/Brighton-FTC/FTC-2023/commit/780daa9eb4fae6e9e44959289cbfc7afb3590e55
Fixed inputs and how to get servos: https://github.com/Brighton-FTC/FTC-2023/commit/76b1290ee855c08fa74baf437403b611c5d88fd3

Full code: https://github.com/Brighton-FTC/FTC-2023/blob/workspace_sbottingota/TeamCode/src/main/java/org/firstinspires/ftc/teamcode/opModes/test/SBOTTINGOTATestGrabber.java

**Comment by sbottingota** [comment]
@WebCoder49, do you know what angle the grabber servo is at for the open/closed positions? (Right now I have them set as 0 deg and 90 deg respectively, but reality probably won't be so simple.) And if not, how can I find out?

**Comment by WebCoder49** [comment]
> @WebCoder49, do you know what angle the grabber servo is at for the open/closed positions for each of the teams? (Right now I have them set as 0 deg and 90 deg respectively, but reality probably won't be so simple.) And if not, how can I find out?

@sbottingota 
No; I am not sure right now. Please ask in Teams/in person tomorrow. Also, it may be easier to test the grabber and get a position in encoder counts rather than degrees.
